packages/matter-node.js/src/environment/ProcessManager.ts

```diff
@@ -63,6 +69,10 @@ export class ProcessManager implements Destructable {
         return this.env.vars.get("runtime.exitcode", true);
     }
 
+    protected get hasUnhandledErrorSupport() {
+        return this.env.vars.get("runtime.unhandlederrors", true);
```

Comment:
Wer should add that to the in-code docs from NodeJsEnvironment to nbe documented for now? WDYT?

Reply:
added